File: docs/snippets/csharp/getting-started/install_verify.cs
using Xberg;

var version = XbergClient.GetVersion();
Console.WriteLine($"Xberg version: {version}");

var result = (await XbergConverter.ExtractAsync(ExtractInput.FromUri("document.pdf"), ExtractionConfig.Default())).Results[0];
Console.WriteLine($"Extraction successful: {result.Success}");

File: docs/snippets/elixir/getting-started/install_verify.exs
```elixir title="Elixir"
# Verify Xberg is installed and working by extracting a document
{:ok, output} = Xberg.extract(input: %Xberg.ExtractInput{kind: :uri, uri: "sample.pdf"}, config: nil)
result = List.first(output.results)
IO.puts("Installation verified! Extracted #{String.length(result.content)} characters")
```
